BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -324,7 +324,7 @@ public final class ConvivaAnalytics: NSObject {
     }
 
     private func buildDynamicContentMetadata() {
-        if !player.isLive {
+        if !player.isLive && player.duration.isFinite {
```

Comment:
We observed that issue during development and this issue will be resolved in the next `SDK` version. However, the new value will be `0` then and not `NAN` anymore. Not sure anymore if a duration of `0` is valid for conviva.

Reply:
thanks @stonko1994 for the input. We can check about setting value to `0` with realeyes team. I believe this fix may still be relevant for customer's using the older SDK versions. What do you think?

Reply:
Yes definitely. Either we keep this check or we would need to increase the minimum version of the SDK required.